data/files/validation_examples/703.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ratings-java-svc
+  namespace: ratings-java
+  labels:
+    app: ratings
+    service: ratings-svc
+spec:
+  ports:
+  - port: 9080
+    targetPort: 8080
```

Comment:
Is this example creating a warning/error? I understand that this example is valid, right? Shouldn't we modify this to make it fail? Afaik, all the other examples creates validations in order to make the user see where the problem is.
Perhaps understanding wrong the validation.